libs/openchallenges/team/src/lib/team.component.html

```diff
@@ -14,34 +14,82 @@ <h2>Meet Our Team</h2>
     <div class="member-card">
       <img class="member-pic" [src]="(thomas$ | async)?.url" alt="Thomas Schaffter" />
       <h3 class="card-name">Thomas Schaffter</h3>
-      <p class="card-roles">Project lead, Architect, Backend developer</p>
+      <p class="card-roles">Project owner, Architect, Lead full stack engineer</p>
+      <div>
+        <a href="https://www.linkedin.com/in/tschaffter/" rel="noopener" target="_blank"
+          ><img class="social-media-logo" src="assets/images/linkedin-logo.svg" alt="LinkedIn"
+        /></a>
+        <a href="https://github.com/tschaffter" rel="noopener" target="_blank"
+          ><img class="social-media-logo" src="assets/images/github-logo.svg" alt="GitHub"
+        /></a>
+      </div>
     </div>
     <div class="member-card">
-      <img class="member-pic" [src]="(maria$ | async)?.url" alt="Maria Diaz" />
-      <h3 class="card-name">Maria Diaz</h3>
-      <p class="card-roles">Backend developer</p>
+      <img class="member-pic" [src]="(verena$ | async)?.url" alt="Verena Chung" />
+      <h3 class="card-name">Verena Chung</h3>
+      <p class="card-roles">Data analyst, Frontend engineer</p>
```

Comment:
```suggestion
      <p class="card-roles">Frontend engineer</p>
```

I'm okay with my title just being "frontend engineer" 👍🏼